sites/footers/old-footers-en.html

```diff
@@ -7,12 +7,85 @@
 	],
 	"altLangPage": "old-footers-fr.html",
 	"secondlevel": false,
-	"dateModified": "2025-06-12",
+	"dateModified": "2025-08-12",
 	"share": "true"
 }
 ---
 <div class="wb-prettify all-pre hide"></div>
 
+<h2>Version 9.0 (deprecated)</h2>
```

Comment:
We haven't deprecated that version yet (that requires a major version).

```suggestion
<h2>Version 9.0</h2>
```